src/components/heroHeader.tsx

```diff
@@ -0,0 +1,59 @@
+import { Box, Flex, Button, Heading, Text } from "@chakra-ui/react";
+
+const HeroHeader = () => {
+  return (
+    <Flex w="100%" h="100%">
+      <Box
+        bgImage="url('Bounded_Hero.png')"
+        bgSize="cover"
+        bgPosition="center"
+        h="600px"
+        flex="3"
+        display="flex"
+        flexDir="column"
+        justifyContent="center"
+      >
+        <Heading size="xl" color="white" className="font-bold">
+          Your Hero Text
+        </Heading>
+
+        <Text pt={2}>Your description here</Text>
+
+        <Button w="120px" bgColor="brand.100" size="lg" mt={4}>
+          Call to Action
+        </Button>
+      </Box>
+
+      <Flex flex="1">
+        <Box
+          h="100%"
+          bgImage="url('Collapsed_Hero_1.png')"
+          bgSize="cover"
+          bgPosition="center"
+          mb={4}
+          display="flex"
+          alignItems="center"
+        >
+          <Heading size="md" p={2} transform="rotate(-90deg)">
+            Right Bar Title
+          </Heading>
+        </Box>
+
+        <Box
+          h="100%"
+          bgImage="url('Collapsed_Hero_2.png')"
+          bgSize="cover"
+          bgPosition="center"
+          display="flex"
+          alignItems="center"
+        >
+          <Heading size="md" color="white" p={2} transform="rotate(-90deg)">
```

Comment:
We should use the theme colors here I think even if it's just white.

```suggestion
          <Heading size="md" color="brand.500" p={2} transform="rotate(-90deg)">
```

I'm not 100% sure I set up the theme correctly here (used to Material UI themes) but this seems to be the correct syntax for the time being. The reason why we aren't using Material UI anyway is I've had performance issues with it on larger projects.

Reply:
Just for context actually, here is the theme file:
https://github.com/PXB-Media/rivals-league/blob/d3c1bca18449bec693d0c8552a41cd73db957096/src/lib/theme.ts#L16